internal/model/model.go

```diff
@@ -1489,6 +1503,71 @@ func (m *Model) BringToFront(folder, file string) {
 	}
 }
 
+// Returns current folder error, or an empty string if there isn't an error.
+func (m *Model) FolderError(id string) string {
```

Comment:
Method name sounds to me like it either just returns an error set somewhere else, or possibly sets the folder as errored... Maybe something like `CheckFolderHealth` or similar, returning an actual `error` that may be nil?